Main.php

```diff
@@ -7,10 +7,10 @@ class Main extends \Idno\Common\Plugin
 
         function registerPages()
         {
-            \Idno\Core\Idno::site()->addPageHandler('/comic/edit/?', '\IdnoPlugins\Comic\Pages\Edit');
-            \Idno\Core\Idno::site()->addPageHandler('/comic/edit/([A-Za-z0-9]+)/?', '\IdnoPlugins\Comic\Pages\Edit');
-            \Idno\Core\Idno::site()->addPageHandler('/comic/delete/([A-Za-z0-9]+)/?', '\IdnoPlugins\Comic\Pages\Delete');
-            \Idno\Core\Idno::site()->addPageHandler('/comic/([A-Za-z0-9]+)/.*', '\Idno\Pages\Entity\View');
+            \Idno\Core\Idno::site()->routes()->addRoute('/comic/edit/?', '\IdnoPlugins\Comic\Pages\Edit');
+            \Idno\Core\Idno::site()->routes()->addRoute('/comic/edit/([A-Za-z0-9]+)/?', '\IdnoPlugins\Comic\Pages\Edit');
```

Comment:
You'll want to use the new UUID format, there's a definition in core for it as a shorthand (but basically you need an `\-` in the regex.

Reply:
like in https://github.com/idno/known/blob/543ff0e35c8576abff8637d711b056afe08474c1/Idno/Core/PubSubHubbub.php#L155

or a literal escaped UUID?

The fact nobody that uses this has offered input makes me think it might not be the most strategic use of time, but 🤷 . What do I know?